docs/developer/apidocs/blas1.rst

```diff
@@ -63,7 +63,7 @@ sum
 swap
 ----
 .. doxygenfunction:: KokkosBlas::swap(execution_space const& space, XVector const& X, YVector const& Y)
-.. doxygenfunction:: KokkosBlas::swap(XVector const& X, YVector const& Y)
+.. doxygenfunction:: KokkosBlas::swap(const XVector&, const YVector&)
```

Comment:
Why does the other overload have names for parameters but not this one?

Reply:
I think this was likely a copy-paste slip-up from one of the sphinx warning messages. The names are not required.